docs/trace_service_features.md

```diff
@@ -6,16 +6,16 @@ Emoji Legend
-| Email           | Ruby            | :100:                     | :100:                | :100:                | :construction:          | :construction: | :construction: |
-| Feature Flag    | Erlang / Elixir | :100:                     | :construction:       | :construction:       | :construction:          | :construction: | :construction: |
-| Frontend        | JavaScript      | :100:                     | :100:                | :100:                | :construction:          | :100:          | :100:          |
-| Payment         | JavaScript      | :100:                     | :100:                | :100:                | :construction:          | :construction: | :construction: |
-| Product Catalog | Go              | :100:                     | :construction:       | :100:                | :construction:          | :construction: | :construction: |
-| Recommendation  | Python          | :100:                     | :100:                | :100:                | :construction:          | :construction: | :construction: |
-| Shipping        | Rust            | :construction:            | :100:                | :100:                | :100:                   | :construction: | :construction: |
+| Service         | Language        | Instrumentation Libraries | Manual Span Creation | Span Data Enrichment | RPC Context Propagation | Span Links | Baggage        |
+|-----------------|-----------------|---------------------------|----------------------|----------------------|-------------------------|------------|----------------|
+| Ad              | Java            | :100:                     | :100:                | :100:                | :no_bell:               | :no_bell:  | :no_bell:      |
+| Cart            | .NET            | :100:                     | :construction:       | :100:                | :no_bell:               | :no_bell:  | :no_bell:      |
+| Checkout        | Go              | :100:                     | :100:                | :100:                | :no_bell:               | :no_bell:  | :no_bell:      |
+| Currency        | C++             | :no_bell:                 | :100:                | :100:                | :100:                   | :no_bell:  | :no_bell:      |
+| Email           | Ruby            | :100:                     | :100:                | :100:                | :no_bell:               | :no_bell:  | :no_bell:      |
+| Feature Flag    | Erlang / Elixir | :100:                     | :construction:       | :construction:       | :no_bell:               | :no_bell:  | :no_bell:      |
+| Frontend        | JavaScript      | :100:                     | :100:                | :100:                | :no_bell:               | :100:      | :100:          |
+| Payment         | JavaScript      | :100:                     | :100:                | :100:                | :no_bell:               | :no_bell:  | :construction: |
```

Comment:
We already have the code deployed and working that checks for baggage in the payment service.  This should be 💯 for the baggage category.